widget/entry.go

```diff
@@ -288,6 +273,8 @@ func (e *Entry) FocusGained() {
 	e.setFieldsAndRefresh(func() {
 		e.dirty = true
 		e.focused = true
+
+		log.Println("FOCUS")
```

Comment:
Found a left-over.

Reply:
How embarrassing! Fixed

Reply:
The import was accidentally left in leading to a compile error unfortunately

Reply:
Apologies, I should stop doing code commits on the bus!

Reply:
Relatable. No problem

Reply:
Different kind of bus factor, and we're all in for the ride 😉